docs/components.md

```diff
@@ -57,6 +57,32 @@ Setting `exposedEvents` to `true` rather than an array will expose all events.
 
 Additional events can also be exposed by passing in an array of event names to `exposedEvents` on the component options object.
 
+Events on components are namespaced when exposed. This means the events are bound to only the model of the component regardless of what level in the applicaton it is being used.
+
+To respond to events on a component's model, a parent or ancestor application/component can listen to the event on the model but will have to scope the listening action to the component. Custom events on the component's model must be exposed to its ancestor via the `exposedEvents` object.
```

Comment:
In the below example can you add the code for the component where `exposedEvents` is defined?